acceptance-tests/tests/create.spec.ts

```diff
@@ -16,22 +15,27 @@ const FOLDERS = {
   },
   websiteTheme: {
     folder: 'website-theme',
+    name: 'website-theme',
```

Comment:
I was in this file and noticed these were inconsistently used and used made it consistent.  This is pretty much unrelated to the rest of the changes.